tests/js/server/recovery/foxx-directories.js

```diff
@@ -62,14 +62,20 @@ function runSetup () {
   // we need to wait long enough for the DatabaseManagerThread to 
   // physically carry out the deletion
   let path = fs.join(appPath, 'UnitTestsRecovery2');
+  let gone = false;
   let tries = 0;
-  while (++tries < 30) {
+  throw new Error("blarg");
```

Comment:
WTF?

Reply:
intended to detect whether this PR is effective.

Reply:
meanwhile removed.